### ISSUE TITLE
Feature Request: Hotkey to react to last message.

### ISSUE THREAD

**Issue by MicahZoltu** [RocketChat/Rocket.Chat#9412]
Slack has a feature where if you press `ctrl+shift+/` it will open the reaction fly-out on the last message with the keyboard focus being in the reaction search box.  This allows a user to quickly hit the hotkey and then find an appropriate emoji (either by typing or via mouse) to apply as a reaction to the previous message.

Note that this is notably different from `+:reaction:` because the message the reaction is attached to is determined at the moment the hotkey is pressed, rather than when the emoji is selected.  When using `+:reaction:`, it is very easy in a high throughput channel to accidentally apply the reaction to the wrong message because it takes non-zero time to choose/type a reaction and if anyone sends a message between the one you want to react to and you submitting the reaction it will be applied incorrectly.  With the hotkey, the user is able to visually verify what they are reacting to _before_ submitting the reaction, thus vastly reducing the rate of misplaced reactions.  This is particularly meaningful when you are not very fast at picking your reaction (e.g., you have to look for it).

The advantage of a hotkey to open the reaction fly-out vs the mouse is that the user can remain with both hands on the keyboard for the entire exchange, which is valuable when engaging in extended conversation.

**Comment by rafaelrestolho12** [comment]
Any update on this?
